src/brainseg/create_fh_seg.py

```diff
@@ -0,0 +1,121 @@
+import os
+import sys
+import subprocess
+import nibabel as nib
+import numpy as np
+import scipy.ndimage
+import shutil
+
+def run_synthseg(conda_env, input_nifti, output_nifti, synthseg_script, synthseg_dir):
+    python_bin = os.path.join(conda_env, "bin", "python")
+    env = os.environ.copy()
+    env["PYTHONPATH"] = synthseg_dir
+
+    subprocess.run(
```

Comment:
I _believe_ what we should rather do in this subprocess call is

- sourcing the correct conda
- activate the synthseg conda environment
- then run synthseg